src/app/swap/get-active-loopd.ts

```diff
@@ -0,0 +1,33 @@
+import { getActiveOnchainLnd } from "@services/lnd/utils"
+import { SwapErrorNoActiveLoopdNode } from "@domain/swap/errors"
+import { BTC_NETWORK, getSwapConfig } from "@config"
+
+export const getActiveLoopd = (): LoopdConfig => {
+  const activeNode = getActiveOnchainLnd()
+  if (activeNode instanceof Error) throw SwapErrorNoActiveLoopdNode
+  switch (activeNode.macaroon) {
```

Comment:
it's surprising to do the switch on the macaroon. 

this is the admin macaroon from lnd1/lnd2? not loop?

Reply:
I basically need to know the name of the active onchain LND instance, specifically if its called LND1 or LND2. When I call `getActiveOnChainLnd()` it returns data about the macaroons and tls certs but it does not specifically tell me its name (LND1 or LND2) . This is what it return:

<img width="719" alt="Screen Shot 2022-08-31 at 9 56 12 PM" src="https://user-images.githubusercontent.com/1273575/187822844-1dc8e82f-94d9-42b7-8bcd-91129995fe33.png">

After I know the name of the LND instance I can get its corresponding LOOP config based on the naming convention `LND1_LOOP_CONFIG` or `LND2_LOOP_CONFIG`. Any ideas on how to get the LND instance name?

Reply:
added name to LNDParams